toolset/benchmark/benchmarker.py

```diff
@@ -963,6 +964,8 @@ def __init__(self, args):
 
     # setup results and latest_results directories 
     self.result_directory = os.path.join("results", self.name)
+    if args['clean'] == 'results':
+        shutil.rmtree(self.result_directory)
```

Comment:
I'm fine with adding `--clean`.

Broader question: does it make sense for `--clean` to also nuke the installs directory?